google_guest_agent/main.go

```diff
@@ -115,11 +115,12 @@ func runUpdate() {
 	}
 
 	var wg sync.WaitGroup
-	var mgrs []manager
-	if runtime.GOOS == "windows" {
-		mgrs = []manager{newWsfcManager(), &addressMgr{}, &winAccountsMgr{}}
-	} else {
-		mgrs = []manager{&clockskewMgr{}}
+	mgrs := []manager{newWsfcManager(), &addressMgr{}}
```

Comment:
Do we want to enable wsfcManager?

Reply:
i assumed so since it was before.....

Reply:
I meant for Linux

Reply:
oh, duh. no we don't, moved into the windows block.